.github/workflows/run-tarpaulin.yml

```diff
@@ -63,7 +63,17 @@ jobs:
     - name: Run tarpaulin
       run: docker exec $(cat container_id.txt) sh -c "RUST_LOG=trace cargo tarpaulin -v --all-features --out Xml"
 
-    - name: Upload report to codecov
+    - name: Upload report to codecov for PR

```

Comment:
When would this trigger if pull_request* is disqualifying?

Reply:
**for PRs**: `Upload report to codecov for PR` will trigger when `startsWith(github.event_name, 'pull_request')`

**for Push**: `Upload report to codecov for push` will trigger when `!startsWith(github.event_name, 'pull_request')`

Reply:
(push is the merge)

Reply:
My bad, saw the ! for the first one as well, somehow.